2.4/root/usr/bin/run-mongod

```diff
@@ -47,9 +47,7 @@ function cleanup() {
   fi
   echo "=> Shutting down MongoDB server ..."
   if [ -f "${MONGODB_PID_FILE}" ]; then
-    kill -2 $(cat ${MONGODB_PID_FILE})
-  else
-    pkill -2 mongod
+    mongod --shutdown $mongo_common_args
```

Comment:
No, if `mongod` is not running this call exits with error code 1 and error message `There doesn't seem to be a server running with dbpath: ...`

It very similar to kill or pkill...